### PR TITLE
Login

### PR DESCRIPTION
**Pull request by FomasTreeman**

### CODE REVIEW

File: src/components/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//             </div>
+//             <TextInput id="password2" required shadow type="password" />
+//           </div>
+//           <Button className="self-end" color="purple">
+//             <Link href="/user/sign-up"> Don&apos;t have an account yet? </Link>
+//           </Button>
+//           <Button type="submit">Sign In</Button>
+//         </form>
+//       </Card>
+//     </main>
```
Comment (camelPhonso) [inline src/components/Form.tsx:29]: Ghost Code ! Who you gonna call?

Can you please delete all of this? 👻

File: src/components/Map.tsx [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 import { initializeMap } from 'utils/mapHelper';
 import { IMapProps } from '../../types/types';
+
 const Map = ({ center, markers, id }: IMapProps) => {
   useEffect(() => {
-    if (typeof window !== 'undefined') {
-      initializeMap({id, center, markers});
-    }
+    if (typeof window === 'undefined') return;
+    initializeMap({ id, center, markers });
```
Comment (camelPhonso) [inline src/components/Map.tsx:10]: nice refactor ✨

File: src/components/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// export default function form({}) {
```
Comment (cameochoquer) [inline src/components/Form.tsx:1]: should this file just be deleted?

File: src/app/sign-in/page.tsx
Comment (camelPhonso) [inline src/app/sign-in/page.tsx:1]: can this please be called log-in so we follow standard convention and avoid confusion later on?

File: src/components/SearchBar.tsx [hunk 4]
```diff
@@ -1,27 +1,26 @@
...
-      initializeSearch()
-        .then((selected) => {
-          setLocation(selected as ILocation);
-        })
-        .catch((error) => {
-          console.error(error);
-        });
-    }
-  }, [location]);
+    if (typeof window === 'undefined') return;
```
Comment (cameochoquer) [inline src/components/SearchBar.tsx:15]: remove comments?

File: tailwind.config.cjs [hunk 5]
```diff
@@ -17,5 +17,5 @@ module.exports = {
     },
   },
   plugins: [require('flowbite/plugin')],
-  darkMode: 'media',
```
Comment (cameochoquer) [inline tailwind.config.cjs:20]: are you sure!?

**Review by cameochoquer — DISMISSED** [review]

File: src/components/SearchBar.tsx (hunk 4 above)
Comment (FomasTreeman) [reply src/components/SearchBar.tsx:15]: unnecessary code atm. so keeping it blanked until its used

File: tailwind.config.cjs (hunk 5 above)
Comment (FomasTreeman) [reply tailwind.config.cjs:20]: yes

Comment (FomasTreeman) [reply tailwind.config.cjs:20]: ?

**Review by cameochoquer — APPROVED** [review]

**Review by camelPhonso — APPROVED** [review]

**Pull request merged**